grails-app/services/org/pih/warehouse/inventory/CycleCountImportService.groovy

```diff
@@ -9,6 +9,17 @@ class CycleCountImportService {
 
     LocalizationService localizationService
 
+    /**
+     * Util function to normalize the default lot number - default lot number can be represented
+     * by either empty string or null. If we compare lot numbers where one of them is empty string
+     * and the other is null, we want them to be treated as equal.
+     * @param lotNumber
+     * @return
+     */
+    private static normalizeLotNumber(String lotNumber) {
```

Comment:
Is the word "normalize" proper here? Won't "standardize" be better?

Reply:
I find them as synonyms, so let me know, and let's here from others what you prefer, so I can eventually adjust it.